### PR TITLE
lint code with black

### PR DESCRIPTION
**Pull request by williballenthin**
(this depends on #4 for GH Actions configuration. please review and merge/reject that first).

Like #6, this introduces a code linter that you can optionally decide to use for dnfile. If you like it, then this PR configures GH Actions to check for violations during CI. 

[black](https://github.com/psf/black) is a very opinionated code linter and formatter. While it uses a style that probably very few people agree with 100.0%, it makes some very good choices and has compelling reasoning for them. Its also developed by the PSF and has pretty widespread use across Python projects. In my projects, I like to use it because it removes my need to decide what a project's style is and why - I just outsource that to black. You may or may not agree with this :-)

The idea here is to enforce a basic set of consistency. If you like it, please merge! If not, no worries!

Note: you can use the command `python -m black -l 120 .` to automatically apply all fixes recommended by black. This is what I do with all my projects before pushing/creating a PR.

### CODE REVIEW

File: src/dnfile/enums.py [hunk 1]
```diff
@@ -61,28 +62,29 @@ def __iter__(self):
...
-    CLR_NATIVE_ENTRYPOINT   = 0x00000010
-    CLR_TRACKDEBUGDATA      = 0x00010000
-    CLR_PREFER_32BIT        = 0x00020000
+    CLR_ILONLY = 0x00000001
+    CLR_32BITREQUIRED = 0x00000002
+    CLR_IL_LIBRARY = 0x00000004
+    CLR_STRONGNAMESIGNED = 0x00000008
+    CLR_NATIVE_ENTRYPOINT = 0x00000010
+    CLR_TRACKDEBUGDATA = 0x00010000
+    CLR_PREFER_32BIT = 0x00020000
```
Comment (williballenthin) [inline src/dnfile/enums.py:75]: I certainly appreciate how you took the time to vertically align the elements in `enum.py`. If I had my way, I'd disable black from changing this; however, this is an example of how black is opinionated and uncompromising: there's not a way to do that. So, you have to decide if its worthwhile to keep the vertical alignment, or the consistency of black.

**Comment by malwarefrank** [comment]
I like black in general, but that vertical alignment helps with readability.  I'll have to think on this one a bit.  Maybe just exclude enums.py

**Comment by williballenthin** [comment]
given that we've got pycodestyle running in CI, i think its ok to use that linter, and to close this PR.

**Comment by malwarefrank** [comment]
Closing for now.  May revisit if future interest

**Pull request closed without merging**